### PR TITLE
RANCHER-1520 Helm values feature-switchers not work properly

### PR DESCRIPTION
**Pull request by OHaimanov**
JIRA: https://folio-org.atlassian.net/browse/RANCHER-1520

### CODE REVIEW

File: vars/folioHelm.groovy [hunk 1]
```diff
@@ -167,76 +167,38 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
...
-//
-//  if (ns.getClusterName() == 'folio-dev') {
-//    moduleConfig << [modSearchDev: "true"]
-//    moduleConfig << [modInventoryStorageDev: "true"]
-//    moduleConfig << [modEntitiesLinksDev: "true"]
-//  }
-//
+  //Allow (in consortia setups) or deny cross tenant requests
+  if (moduleName == 'mod-authtoken' && ns.enableConsortia) {
+    moduleConfig['extraJavaOpts'] += ['-Dallow.cross.tenant.requests=true']
```
Comment (eldiiar-duishenaliev) [inline vars/folioHelm.groovy:189]: We have agreed to have cross tenants requested enabled by default...

**Review by eldiiar-duishenaliev — CHANGES_REQUESTED** [review]
pls review comment

File: vars/folioHelm.groovy (hunk 1 above)
Comment (OHaimanov) [reply vars/folioHelm.groovy]: resolved

**Review by eldiiar-duishenaliev — APPROVED** [review]

**Review by epam-avramenko — APPROVED** [review]

**Review by yaroslavishchenko — COMMENTED** [review]
Everything looks good

**Pull request merged**